openvino_supported_tests.txt

```diff
@@ -0,0 +1,21 @@
+keras-hub/integration_tests
```

Comment:
I think keeping this whitelist in python and not a txt file would be good. This will likely just get stale. Maybe we can just add this to conftest.py by modifying `pytest_collection_modifyitems` if backend == openvino.

Maybe let's just entirely skip `keras_hub/src/layers` for now, and we can do the layer whitelisting as a follow up.